packages/lodestar/src/api/rest/controllers/beacon/state/getValidators.ts

```diff
@@ -27,6 +42,19 @@ export const getStateValidators: ApiController<DefaultQuery, Params> = {
           },
         },
       },
+      querystring: {
+        type: "object",
+        required: [],
+        properties: {
+          indices: {
+            types: "string",
```

Comment:
```suggestion
            type: "array",
```
should allow you to always get array even with single item

Reply:
ahh yeah ok i messed up `types` vs `type` when i had tried this originally.  just pushed a change for this.